i18n/en/docusaurus-plugin-content-docs/current/migrations/how-to-upgrade-hermione-to-7.mdx

```diff
@@ -0,0 +1,137 @@
+import Admonition from "@theme/Admonition";
+
+# How to Upgrade hermione to Version 7.x
+
+<Admonition type="warning">
+If your project is using a hermione version earlier than 4.x, first read "How to Upgrade hermione to Version 4.x," "How to Upgrade hermione to Version 5.x," and "How to Upgrade hermione to Version 6.x."
+
+We recommend upgrading hermione in stages, ensuring each step that all project tests run correctly.
+
+</Admonition>
+
+## What Has Changed? {#what_is_new}
+
+### Major Changes {#major_changes}
+
+#### Dropped Support for Node.JS < 16.x {#dropped_nodejs_less_than_16x}
+
+In this major version, hermione no longer supports versions _Node.JS < 16.x_.
+
+#### Handling Unhandled Rejections {#handle_unhandled_rejection}
+
+When an `Unhandled rejection` error occurs in the master or workers, hermione now performs a graceful shutdown. This means all subsequent tests in the queue will end with the error `Browser request was cancelled`, and hermione itself will terminate with `exit code 1`.
+
+This change is necessary to protect users from writing incorrect tests. Example of a test with an error:
+
+```typescript
+it("test", async ({ browser }) => {
+    await browser.url("https://yandex.ru/search/?text=aaa");
+    expect(browser).toHaveUrlContaining("foo-bar-baz");
+});
+```
+
+A keen observer will notice that I forgot to add `await` before calling `expect`. In this case, the test will immediately pass after navigating to the URL. Only then will an `Unhandled rejection` error appear, which hermione@6 would swallow. But hermione@7 will catch and immediately terminate the test run. Unfortunately, it is impossible to determine from which test the `Unhandled rejection` error originated. For example, a test could be written like this:
+
+```typescript
+it("test2", async () => {
+    new Promise((resolve, reject) => {
+        setTimeout(() => {
+            reject("Something goes wrong");
+        }, 30000);
+    });
+});
+```
+
+This test will immediately pass, and only after 30 seconds (assuming there are still tests in the queue to run) will an `Unhandled rejection` error appear.
+
+Therefore, to find the problematic test, you will need to look at the execution log and find the last completed tests before the error information. The error will look something like this:
+
+```bash
+[13:48:57 GMT+3] Unhandled Rejection in hermione:worker:10830:
```

Comment:
`hermione:worker` -> `testplane:worker`

Reply:
No, in fact this is accurate, because describes migration process from hermione v6 to v7